cocos/game/game.ts

```diff
@@ -1032,6 +1039,12 @@ export class Game extends EventTarget {
         this.resumeByEngine();
     }
 
+    private _onClose () {
+        this.emit(Game.EVENT_CLOSE);
+        // TODO : Release Resources.
+        systemInfo.exit();
```

Comment:
It's strange to add a behavior `exit` to a `Info` class. 
I think `systemInfo` classes should only provide `system informations`, not to control `system`.

Reply:
There is 'close' inside systemInfo to close the window.
And there are messages received from the system, including onClose, onShow, onHide.
So I think systemInfo should be renamed as system.

Reply:
Yep, the name should be considered carefully. `...Info` is not suitable here now.